pkg/util/mount/mount_linux.go

```diff
@@ -944,6 +951,57 @@ func doCleanSubPath(mounter Interface, fullContainerDirPath, subPathIndex string
 	return nil
 }
 
+// cleanCorruptedMnt tears down the corrupted subpath bind mount
+func cleanCorruptedMnt(mounter Interface, fullSubPath string) error {
```

Comment:
add unit test for this?

Reply:
Just `umount` and `remove` operation, it almost the same with `doCleanSubPath`